_pytest/terminal.py

```diff
@@ -64,8 +64,10 @@ def getreportopt(config):
     reportchars = config.option.reportchars
     if reportchars:
         for char in reportchars:
-            if char not in reportopts:
+            if char not in reportopts and char != 'a':
```

Comment:
The code above uses `"` for strings, so it'd be nice if you could do the same everywhere for consistency.